The fix is actually incomplete as there is a linking issue:
```
g++ -o bin/godot.x11.opt.tools.64 platform/x11/godot_x11.x11.opt.tools.64.o platform/x11/context_gl_x11.x11.opt.tools.64.o platform/x11/os_x11.x11.opt.tools.64.o platform/x11/key_mapping_x11.x11.opt.tools.64.o main/libmain.x11.opt.tools.64.a modules/libmodules.x11.opt.tools.64.a bin/tests/libtests.x11.opt.tools.64.a drivers/libdrivers.x11.opt.tools.64.a drivers/libdrivers1.x11.opt.tools.64.a drivers/libdrivers2.x11.opt.tools.64.a drivers/libdrivers3.x11.opt.tools.64.a drivers/libdrivers4.x11.opt.tools.64.a tools/libtool.x11.opt.tools.64.a scene/libscene.x11.opt.tools.64.a servers/libservers.x11.opt.tools.64.a core/libcore.x11.opt.tools.64.a -lX11 -lXinerama -lXcursor -lfreetype -lpulse-simple -lpulse -lGL -lGLU -lpthread -lasound -lz
/usr/bin/ld: drivers/libdrivers2.x11.opt.tools.64.a(opusfile.x11.opt.tools.64.o): undefined reference to symbol 'ogg_sync_pageout'
/lib64/libogg.so.0: error adding symbols: DSO missing from command line
```

--- a/drivers/ogg/SCsub
+++ b/drivers/ogg/SCsub
@@ -6,5 +6,5 @@ ogg_sources = [
 	"ogg/framing.c",
 ]
 
-if env['theora'] != "yes" or env['use_theoraplayer_binary'] != "yes":
+#if env['theora'] != "yes" or env['use_theoraplayer_binary'] != "yes":
 	env.drivers_sources+=ogg_sources

--- a/drivers/vorbis/SCsub
+++ b/drivers/vorbis/SCsub
@@ -34,5 +34,5 @@ sources_lib = [
 
 env.drivers_sources += sources
 
-if env['theora'] != "yes" or env['use_theoraplayer_binary'] != "yes":
+#if env['theora'] != "yes" or env['use_theoraplayer_binary'] != "yes":
 	env.drivers_sources += sources_lib

